package.json

```diff
@@ -23,6 +24,7 @@
   "devDependencies": {
     "all-contributors-cli": "^5.4.0",
     "bs-platform": "^4.0.0",
+    "live-server": "^1.2.0",
```

Comment:
Nice! I've always used `http-server`, but this one is nice as it has the live-reload capabilities.